x-pack/plugins/observability_ai_assistant/public/components/chat/chat_header.tsx

```diff
@@ -105,17 +126,47 @@ export function ChatHeader({
             }}
           />
         </EuiFlexItem>
-        <EuiFlexItem
-          grow={false}
-          className={showLinkToConversationsApp ? chatActionsMenuWrapper : ''}
-        >
-          <ChatActionsMenu
-            connectors={connectors}
-            conversationId={conversationId}
-            disabled={licenseInvalid}
-            showLinkToConversationsApp={showLinkToConversationsApp}
-            onCopyConversationClick={onCopyConversation}
-          />
+
+        <EuiFlexItem grow={false}>
+          <EuiFlexGroup gutterSize="s">
+            {flyoutWidthMode && onToggleFlyoutWidthMode ? (
+              <>
+                <EuiFlexItem grow={false}>
+                  <EuiButtonIcon
+                    aria-label={i18n.translate(
+                      'xpack.observabilityAiAssistant.chatHeader.euiButtonIcon.navigateToConversationsLabel',
+                      { defaultMessage: 'Navigate to conversations' }
```

Comment:
It's not obvious to me what this button does, wonder if we should add a tooltip to it ?

Reply:
I was about to say the same thing. I intuitively understand 3 out of 4 icons but not the  "Navigate to conversations"  icon.

![image](https://github.com/elastic/kibana/assets/209966/b69e14ed-91c4-4de4-882a-162b05706166)


I think it's especially important to make it clear what the "Navigate to conversations" icon does because it performs an actual page navigation.

Reply:
Yes, I considered that, was wondering if it wouldn't make the UI too busy. But yeah, can be added.

Reply:
Done!